### DISCUSSION TITLE
如何卸载

### DISCUSSION THREAD

**Discussion by xyhcode** [chaitin/SafeLine#328] [Q&A] [answered]
### 反馈内容

如何卸载这个长亭雷池

**Comment by luweijun1992** [comment, marked as answer]
如下：
```
cd 安装目录
docker compose down
cd ..
rm -rf 安装目录
```

**Comment by Lorna0** [comment]
> 如下：
> 
> ```
> cd 安装目录
> docker compose down
> cd ..
> rm -rf 安装目录
> ```

确实是这样操作。我们找个时间写到官网文档里，issue 先关了。

**Comment by ChenXiaoMian** [comment]
```
// 显示当前正在运行的容器
docker ps

// 停止所有命名开头为 ”safeline-“ 的容器
docker stop safeline-redis &&
docker stop safeline-mgt-api &&
docker stop safeline-mario &&
docker stop safeline-tengine &&
docker stop safeline-detector
```

**Comment by zclaiqcc** [comment]
> ```
> // 显示当前正在运行的容器
> docker ps
> 
> // 停止所有命名开头为 ”safeline-“ 的容器
> docker stop safeline-redis &&
> docker stop safeline-mgt-api &&
> docker stop safeline-mario &&
> docker stop safeline-tengine &&
> docker stop safeline-detector
> ```

这个不完整，还有一个 safeline-ce 的 docker network。建议 docker compose down
